### DISCUSSION TITLE
Missing a feature overview

### DISCUSSION THREAD

**Discussion by leo-hstone** [Zettlr/Zettlr#3227] [General]
Zettlr has grown in features. Some of them (mermaid, stats, snippets syntax, etc) are only hidden deep in the docs or are not mentioned at all. 

I propose to include a feature overview in some form.  Some options I can think of:
1. Include a full, well sorted list with links in one of the first pages of the docs
2. Build a  "demo project" which displays **all** feautures "live" in zettlr. This could be hosted in a new repo like Zettlr/demo (for users to just download the zip) or become part of the tutorial workspace.
3. Somehow include it in the recently discussed repo for collecting user customizations.

My personal favorite is option 1 and 2 in combination. The docs for users who are interested in zettlr as a quick overview and the demo project for some hands on examples and inspiraiton.

Regardless of the chosen implementation, this would be a topic where I would like to contribute! But we should decide first where 😄

**Comment by nathanlesage** [comment]
Sounds like a good idea, but I hope it is okay for me to only provide advice, not really contribute to such a thing (I can certainly create such a repository under the `Zettlr` namespace). The reason is that I'm realizing that the Zettlr community is growing insanely in the last days, and so are the new bug reports coming in, and I somehow have to write a dissertation in between …

**Reply by rialistic** [reply]
I could contribute to point 1. by suggesting edits to the texts and the stucture. We could draft a doc of the features first, check how it aligns with the current documentation and Getting started advice and the suggest changes to the public version.

Building a demo sounds great, but goes beyond my skills and sounds resource-consuming. I could help in doing tutorials with screenshots and short annotated screencapture videos. I've been making internal guidelines for software use for research institutes and so.

For example[ Tango](https://www.tango.us/  ) software makes it quite easy to do manuals with screenshots and links.

**Reply by leo-hstone** [reply]
Hendrik, this is perfectly fine! I want to contribute by actually creating the overview/demo, but I need you to approve the concept and "put it in the right place". Which option do you think is best suited for new users to discover and learn zettlr's features?

**Reply by leo-hstone** [reply]
@rialistic Tango sounds like a cool project, but I think Zettlr does not require complex tutorials, since every feature itself is quite simple. See my comment below for a few details how I imagined the feature overview :)

**Reply by nathanlesage** [reply]
> but I need you to approve the concept and "put it in the right place".

In this regard, I would argue it makes sense to just start something, draw something up, and then discuss this among the community. In the end, I know each and every miniscule feature of the app, so I'm the worst person to judge "is this good explained?" 😄 

Probably, it might make sense for you to just do something where you think that it might help, and then have it the community discuss this on Discord and/or here on GitHub discussions, iteratively improve it, and then we can make it nice and shiny, and finally integrate it into the official channels! What about that?

**Comment by leo-hstone** [comment]
A few additions how I imagine the proposed options:

**The docs:** I imagine the feature overview in the docs like a long list with a small section to every feature. Every section contains a small description (2 to 3 sentences), a link to the detailed explaination and for example a gif image with an animation visualizing the feauture. 

**The demo project**: As a new user, you donwload the demo project, which is essentially one workspace. In this workspace you could have some starting page explaining the most basics (What is markdown, what are links). From there you could have one page about every feature: For example on about mermaid diagrams which has some actual diagrams in it. As new user, you can just explore the diagrams and the code behind in Zettlr itself.
